### PR TITLE
Implement evaluation mode for model evaluations

### PR DESCRIPTION
**Pull request by FelixLehn**
### Summary
In PART 1 of the tutorial, the classification and regression models use the model.train() setting but not the model.eval() setting with torch.no_grad().

### Details and comments
The model.eval() setting with torch.no_grad() is used in PART 2, which makes sense as no test data is used in PART 1, but for the overall understanding and the PyTorch alignment, shouldn't it be there, too?

### CODE REVIEW

**Comment by FelixLehn** [comment]
You're right. It is otherwise explained twice.

**Review by adekusar-drl — APPROVED** [review]
Thank you.
